### PR TITLE
874593 - Prepare UG documentation for new features in the Maui Navigation drawer

### PR DESCRIPTION
**Pull request by Arunachalam3641**
Prepare UG documentation for new features in the Maui Navigation drawer

### CODE REVIEW

File: MAUI/NavigationDrawer/Swipe-Gesture.md [hunk 1]
```diff
@@ -39,4 +39,35 @@ this.Content = navigationDrawer;
 
 {% endhighlight %}
 
+{% endtabs %}
+
+## Swipe Sensitivity
+
+The TouchThreshold property in the [SfNavigationDrawer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.NavigationDrawer.html) can expand the swipe region.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/NavigationDrawer/Swipe-Gesture.md:46]: mention the default value

File: MAUI/NavigationDrawer/Swipe-Gesture.md [hunk 2]
```diff
@@ -39,4 +39,35 @@ this.Content = navigationDrawer;
...
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<navigationdrawer:SfNavigationDrawer x:Name="navigationDrawer">
+    <navigationdrawer:SfNavigationDrawer.DrawerSettings>
+        <navigationdrawer:DrawerSettings TouchThreshold="80">
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/NavigationDrawer/Swipe-Gesture.md:54]: give threshold higher than the default value

File: MAUI/NavigationDrawer/Swipe-Gesture.md (hunk 1 above)
Comment (Arunachalam3641) [reply MAUI/NavigationDrawer/Swipe-Gesture.md]: Yes, mentioned the default value.

File: MAUI/NavigationDrawer/Swipe-Gesture.md (hunk 2 above)
Comment (Arunachalam3641) [reply MAUI/NavigationDrawer/Swipe-Gesture.md]: Yes, provided a higher value.

**Review by naveenkumar-sanjeevirayan — APPROVED** [review]

**Pull request merged**